infra/terraform/test-org/org/locals.tf

```diff
@@ -661,8 +661,7 @@ locals {
       short_name      = "deploy-java-gke"
       org             = "GoogleCloudPlatform"
       description     = "Deploy a Legacy Java App GKE"
-      owners          = ["donmccasland"]
-      groups          = [local.jss_common_group]
```

Comment:
`donmccasland` is part of `dee-platform-ops`, so `donmccasland` will still have ownership.